File: windowed-wordcount-stream/src/main/java/io/pivotal/demo/repositories/WordCountWindowedRepo.java
package io.pivotal.demo.repositories;

import io.pivotal.demo.domain.WordCountWindowedCompsiteKey;
import io.pivotal.demo.domain.WordCountWindowed;

import org.springframework.data.gemfire.repository.GemfireRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface WordCountWindowedRepo extends GemfireRepository<WordCountWindowed, WordCountWindowedCompsiteKey> {
}

File: web-ui/src/main/kotlin/io/pivotal/demo/repositories/WordCountWindowedRepo.kt
package io.pivotal.demo.repositories

import io.pivotal.demo.domain.WordCountWindowed
import io.pivotal.demo.domain.WordCountWindowedCompsiteKey
import org.springframework.data.gemfire.repository.GemfireRepository
import org.springframework.stereotype.Repository

@Repository
interface WordCountWindowedRepo : GemfireRepository<WordCountWindowed, WordCountWindowedCompsiteKey>
